### ISSUE TITLE
SE is removing OpenID

### ISSUE THREAD

**Issue by Bhargav-Rao** [nathan-osman/go-sechat#2]
https://meta.stackexchange.com/q/307647/311398

It looks like the chat library is using OpenID for authentication https://github.com/nathan-osman/go-sechat/blob/563300db9224ed01797dfe1f89f489e8eb9dd5ba/auth.go#L95. 

Would this library be updated to use the SE login directly?

**Comment by nathan-osman** [comment]
Yes, the package will be updated once the new auth flow is in place.
